playground/assets/__tests__/without-trailing-slash/without-trailing-slash-assets.spec.ts

```diff
@@ -0,0 +1,390 @@
+import { describe, expect, test } from 'vitest'
```

Comment:
maybe you could replace the entire contents of this file with `import '../assets.spec'` to avoid the duplication?

Reply:
I'll have a look into this, but might not be able to do so before in two weeks.